.github/workflows/ci-additional.yaml

```diff
@@ -40,8 +40,8 @@ jobs:
         os: ["ubuntu-latest"]
         env:
           [
-            "py37-bare-minimum",
-            "py37-min-all-deps",
+            "py38-bare-minimum",
+            "py38-min-all-deps",
             "py38-all-but-dask",
             "py38-flaky",
```

Comment:
should these be upgraded to `py39`? And once it is actually possible, to `py310`?

Reply:
I agree. I added some comments to clarify this.